drivers/flash/flash_stm32.c

```diff
@@ -157,8 +157,7 @@ static void flash_stm32_flush_caches(const struct device *dev,
 		regs->ACR |= FLASH_ACR_DCEN;
 	}
 #elif defined(CONFIG_SOC_SERIES_STM32F7X)
-	SCB_InvalidateDCache_by_Addr((uint32_t *)(CONFIG_FLASH_BASE_ADDRESS
-						  + offset), len);
+	SCB_InvalidateDCache_by_Addr((uint32_t *)(FLASH_STM32_BASE_ADDRESS
 #endif
```

Comment:
missing `offset` and `len`?

Reply:
yes, that will be rebased as soon as te https://github.com/zephyrproject-rtos/zephyr/pull/67989 is merged